@hubblecommerce/hubble/core/components/detail/ProductDetailBuyboxOptions.vue

```diff
@@ -44,6 +44,11 @@ export default {
             type: Object,
             required: true,
         },
+        dataOptions: {
+            type: Array,
+            required: false,
+            default: () => []
+        },
```

Comment:
Not necessary if comment in buxbox-slot.vue is done

Reply:
Fixed in [27f554c](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/27f554c1061916e8fef173ff88e0efc1db04ef0e)